### PR TITLE
Update zenodo for 6.0

### PR DESCRIPTION
**Pull request by nabobalis**
👋 Hello New contributors!
I am updating the list of authors for the zenodo entry (see https://zenodo.org/record/3779284 for a previous example).
I have mentioned you because you are a new contributor since sunpy 6.0.

If you would like, please leave a comment below with:

The preferred name you would like to be listed as
Your ORCID
Your affiliation

I will then manually add this to the information.
Note that none of this is compulsory!

@Deus1704 
@ViciousEagle03 
@ahmedhosssam 
@Mubin17
@yeppAniket 
@GillySpace27 
@codeblech 
@neutrinoceros 
@NucleonGodX 
@Ghaithq 
@sageyu123 
@enthusiastic2003 
@abastola0 
@grahamasam 
@RehanChalana 

TODO: 
- [x] Add the details
- [x] Update the mailmap with extra information
- [x] Update the release notes if names change
- [x] Recreate the zenodo file from fresh

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -162,9 +172,7 @@
             "name": "Rajul Srivastava"
         },
         {
-            "name": "Shane Maloney",
-            "orcid": "0000-0002-4715-1805",
-            "affiliation": "Trinity College Dublin / Dublin Institute for Advanced Studies"
+            "name": "Samuel Bennett"
```
Comment (Cadair) [inline .zenodo.json:175]: @CyclingNinja you got an orcid?
```suggestion
            "name": "Samuel Bennett",
            "affiliation": "Aperio Software Ltd."
```

**Comment by codeblech** [comment]
Hello @nabobalis 

my details are ..
NAME: Yash Malik
ORCID: 0009-0005-7611-034X
Affiliation: None

**Comment by Deus1704** [comment]
Thanks @nabobalis!
Preferred name: Jayraj Dulange
ORCID: 0009-0003-2993-7382
Affiliation: Indian Institute of Technology Gandhinagar: Gandhinagar, IN

**Comment by GillySpace27** [comment]
Thank you!
preferred name: Chris R. Gilly
Your ORCID: 0000-0003-0021-9056
Your affiliation: Southwest Research Institute

**Comment by Ghaithq** [comment]
Thanks @nabobalis.
preferred name: Ghaith Kdimati
ORCID: 0009-0006-8851-7814
affiliation: Cairo University

**Comment by yeppAniket** [comment]
Thanks!
preferred name: Aniket Mishra
Your ORCID: 0009-0008-4790-5604
Your affiliation: None

**Comment by sageyu123** [comment]
Thank you!
preferred name: Sijie Yu
Your ORCID: 0000-0003-2872-2614
Your affiliation: New Jersey Institute of Technology

**Comment by ViciousEagle03** [comment]
Thanks @nabobalis 
Preferred name: Piyush Sharma
ORCID: 0009-0005-1579-5787
Affiliation: Indian Institute of Technology Roorkee: Roorkee, IN

**Comment by abastola0** [comment]
Sure. Here is my Info.

Name: Ashish Bastola
ORCID ID: 0009-0006-4968-1005
Affiliation: Clemson University

Thanks,
Ashish

**Comment by RehanChalana** [comment]
Thank You

Preferred name: Rehan Chalana
ORCID: 0009-0005-1579-5787
Affiliation: Chitkara University , Punjab , IN

**Comment by enthusiastic2003** [comment]
Thanks!

Preferred Name: Sirjan Hansda
ORCID: 0009-0004-0333-3166
Affiliation: Indian Institute of Science, Karnataka, IN

**Comment by neutrinoceros** [comment]
Prefered name: Clément Robert
ORCID: 0000-0001-8629-7068
no affiliation

**Comment by ahmedhosssam** [comment]
Preferred Name: Ahmed Hossam
ORCID: 0009-0007-7321-9109
Affiliation: Zagazig University, Faculty of Computers and Informatics

**Comment by Deus1704** [comment]
I think this is what was intended:
> @neutrinoceros 
<s>@Deus1704</s> ---> @NucleonGodX 
@Ghaithq

**Comment by NucleonGodX** [comment]
> I think this is what was intended:
> 
> > @neutrinoceros
> > @Deus1704 ---> @NucleonGodX
> > @Ghaithq

Oh if this is what was intended,

Preferred Name: Manit Singh 
ORCID: 0009-0002-6031-3810
Affiliation: Netaji Subash University of Technology, IN

**Comment by nabobalis** [comment]
Yes, I needed both but somehow used the same github name twice. Sorry about that.

**Comment by NucleonGodX** [comment]
> Yes, I needed both but somehow used the same github name twice. Sorry about that.

Alright!, thanks @Deus1704 for help.

**Comment by nabobalis** [comment]
@zacharyburnett, do you have a ORCID and an affiliation?

File: .zenodo.json [hunk 2]
```diff
@@ -589,24 +634,22 @@
...
+            "orcid": "0009-0005-7611-034X"
+        },
         {
             "name": "Yukie Nomiya",
             "orcid": "0000-0002-9572-6300",
             "affiliation": "Sapienza University of Rome, Italy"
         },
         {
-            "name": "mridulpandey"
+            "name": "Zach Burnett"
```
Comment (nabobalis) [inline .zenodo.json:647]: @zacharyburnett, do you have a ORCID and an affiliation?

File: .zenodo.json [hunk 3]
```diff
@@ -757,7 +820,12 @@
         {
             "name": "Ratul Das",
             "orcid": "0000-0002-5845-9979",
-            "affiliation": "National Institute of Science Education and Research, Bhubaneswar, Odisha, India"
+            "affiliation": "National Institute of Science Education and Research, Bhubaneswar"
+        },
+        {
+            "name": "Rehan Chalana",
+            "orcid": "0009-0005-1579-5787",
```
Comment (nabobalis) [inline .zenodo.json:827]: Hi @RehanChalana, this is the same ORCID as Piyush Sharma, do you happen to have a your ORCID?

**Review by wtbarnes — APPROVED** [review]

File: .zenodo.json (hunk 2 above)
Comment (zacharyburnett) [reply .zenodo.json:650]: I don't have an ORCID, and you can put my affiliation as STScI. That name is fine, thanks!

**Pull request merged**